operations/system_management_health/Prometheus_Kafka_Error.md

```diff
@@ -0,0 +1,13 @@
+# Prometheus log having Prometheus-Kafka-adapter errors during installation
```

Comment:
This new page is missing an entry in the `operations/` global table of contents: https://github.com/Cray-HPE/docs-csm/tree/c9f51496fa9cc52d82a4ae77280b603f5a923ab5/operations#system-management-health